src/select/parts/styles.scss

```diff
@@ -121,3 +121,33 @@ $checkbox-size: awsui.$size-control;
     color: awsui.$color-text-disabled;
   }
 }
+
+.inline-label-trigger-wrapper {
+  margin-block-start: -7px;
+}
+
+.inline-label-wrapper {
+  margin-block-start: calc(awsui.$space-scaled-xs * -1);
+}
+
+.inline-label {
```

Comment:
for 12px fonts we typically also use `letter-spacing: awsui.$letter-spacing-body-s;` to give better spacing in VR